src/Compilers/CSharp/Portable/Syntax/SyntaxNodeRemover.cs

```diff
@@ -90,7 +90,7 @@ internal SyntaxTriviaList ResidualTrivia
                     }
                     else
                     {
-                        return default(SyntaxTriviaList);
+                        return default;
```

Comment:
These changes in the SyntaxNodeRemover seem unrelated to annotation. Consider reverting so you don't need compiler reviews on this PR.

Reply:
Accident. Reverted.